client/state/sites/selectors.js

```diff
@@ -101,6 +102,27 @@ export function isRequestingSites( state ) {
 }
 
 /**
+ * Returns object describing custom title format
+ * strings for SEO.
+ *
+ * @see client/components/seo/meta-title-editor
+ *
+ * @param  {Object} state  Global app state
+ * @param  {Number} siteId Selected site
+ * @return {Object} Formats by type { frontPage: '%site_name%', posts: '%post_title%' }
+ */
+export const getSeoTitleFormats = ( state, siteId ) => {
+	const site = getSite( state, siteId );
+	const rawFormats = mapKeys(
+		get( site, 'options.advanced_seo_title_formats', {} ),
+		( _, key ) => camelCase( key )
```

Comment:
Are throwaway values also a pattern in PHP? I think it's fine to just call this `value` instead of `_` even if we don't use it here